components/coachindicator/index.css

```diff
@@ -22,6 +22,19 @@
 	block-size: var(--mod-coach-indicator-block-size, var(--spectrum-coach-indicator-block-size));
 	min-block-size: var(--mod-coach-indicator-min-block-size, var(--spectrum-coach-indicator-min-block-size));
 	margin: var(--mod-coach-indicator-gap, var(--spectrum-coach-indicator-gap));
+
+	&.spectrum-CoachIndicator--quiet {
+		--spectrum-coach-indicator-min-inline-size: calc(var(--mod-coach-indicator-quiet-ring-diameter, var(--spectrum-coach-indicator-quiet-ring-diameter)) * 2.75);
```

Comment:
This looks like it's a global token: `--spectrum-coach-indicator-quiet-ring-diameter`.

Maybe we add an alias for this and add the global token back into the theme?